onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -448,6 +471,20 @@
   return Status::OK();
 }
 
+void QNNExecutionProvider::InitQnnGraphConfigs(qnn::QnnGraphConfigsBuilder& configs_builder) const {
```

Comment:
>InitQnnGraphConfigs [](start = 27, length = 19)

move this to qnn_model, only need to pass the htp_graph_finalization_opt_mode_ to qnn_model->ComposeGraph. No need to be a member function also.

Reply:
I think the currently approach is more appropriate because `qnn_model->ComposeGraph` is meant to work with any backend (not just HTP). So, it makes sense to pass in generic `QnnGraph_Configs` instead of the HTP-specific `htp_graph_finalization_opt_mode`. In the future, we may need to provide graph configs for other backends as well.

Reply:
that's true. there could even be variations of htp backend. we've seen htp16 backend that's supported by newer snapdragon revisions.